packages/yoga/src/Popover/web/Popover.jsx

```diff
@@ -0,0 +1,71 @@
+import React, { useState } from 'react';
+import { node, number, oneOf, string } from 'prop-types';
+
+import { Description, PopoverContainer, Title, Wrapper } from './styles';
+
+function Popover({
+  children,
+  title,
+  description,
+  position,
+  width,
+  height,
+  ...props
+}) {
+  const [isPopoverOpen, setIsPopoverOpen] = useState(false);
+
+  function handleShowPopover() {
+    setIsPopoverOpen(true);
+  }
+
+  function handleHidePopover() {
+    setIsPopoverOpen(false);
+  }
+
+  return (
+    <Wrapper {...props}>
+      {isPopoverOpen && (
+        <PopoverContainer position={position} width={width} height={height}>
```

Comment:
How about adding the `role` [dialog](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/dialog_role) here? It seems this is the right role for this case, although we also have the [tooltip](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/tooltip_role) one, we can check which one is better but I think we should add a role here, WDYT?

Reply:
Done!